govcd/vm.go

```diff
@@ -554,6 +554,44 @@ func (vm *VM) HandleInsertMedia(org *Org, catalogName, mediaName string) (Task,
 	})
 }
 
+// Helper function which finds media, calls EjectMedia, waits for task to complete and answer question.
+// Also waits until VM status refreshes - this added as current vCD version has lag in status update.
```

Comment:
The wording "current vCD version" will become unclear after a few months.

Reply:
fixed